main.go

```diff
@@ -102,7 +105,7 @@ func getConfig() (config, error) {
 	}, nil
 }
 
-func createGitHubClient(conf *config) *github.Client {
+func (conf *config) createGitHubClient() *github.Client {
```

Comment:
I'd rather inject the config, so make this `func createGitHubClient(conf *config)  *github.Client`